### PR TITLE
Readme updated

### PR DESCRIPTION
**Pull request by Sddilora**

### CODE REVIEW

**Review by husamettinarabaci — CHANGES_REQUESTED** [review]
Please fix the order of team members.
Team Lead
Team Members...

**Comment by Sddilora** [comment]
oh, okay. I just followed the github's order.

**Review by husamettinarabaci — CHANGES_REQUESTED** [review]
I've written a comment about team members

File: README.md [hunk 1]
```diff
@@ -1,14 +1,71 @@
...
+
+## LICENSE
+
+![GitHub](https://img.shields.io/github/license/UBN-jr/hirob-be-core?style=for-the-badge)
+
+## TEAM
+
+[Hüsamettin ARABACI](https://github.com/husamettinarabaci)
+
+[Dilara DOĞAN](https://github.com/Sddilora)
```
[the remark this answers is not in the document]
Comment (husamettinarabaci) [reply README.md:59]: It is not for us. @Sddilora is the leader of the project and every team member has to know their responsibility and their team leader. Please fix it or you can choose to remove the supporter name from this list. It is a little bit easy way.

**Review by husamettinarabaci — APPROVED** [review]
Good Work

**Pull request merged**